scripts/license_checker.py

```diff
@@ -61,6 +61,7 @@
     "requirements.txt",
     ".vscode",
     ".mypy_cache",
+    "mypy.ini",
```

Comment:
```suggestion
    ".mypy.ini",
```
Since this was renamed.